README.md

```diff
@@ -70,7 +70,7 @@ A resource is some data or process that has an address. It can be anything from
 
 An action possible on some resource. Each action MAY have its own semantics. They MAY be unary, support a hierarchy, be monotone, partial orders, and so on. Actions MAY be general and applicable to many kinds of resource, or tied to a specific one.
 
-For instance, `wnfs/APPEND` is an action for WebNative filesystem paths. The action `wnfs/OVERWRITE` also implies the ability to append. Email has no such tiered relationship. One can `email/SEND`, but there is no concept of a ”super send”.
+For instance, `wnfs/APPEND` is an action for WebNative filesystem paths. The action `wnfs/OVERWRITE` also implies the capacity to append. Email has no such tiered relationship. One can `email/SEND`, but there is no concept of a ”super send”.
```

Comment:
Avoiding using the term in more than the strictest sense (since we've rebound the signifier `ability`)

Reply:
Is this meant to be `capability`?

Reply:
I'm trying to avoid "capability" to keep it narrowly defined in the document. So yes in the colloquial sense, but because it's scoped to actions I'm trying to avoid the term.

I've also been informed that using the term "scope" in a specialized sense in this document is confusing when trying to write an implementation and talking about "auth scope" vs "code scope" versus "project scope", so I'll change that term shortly, too